src/Microsoft.DocAsCode.Common/FileAbstractLayer/EmptyFileReader.cs

```diff
@@ -20,6 +20,9 @@ private EmptyFileReader()
 
         public IEnumerable<RelativePath> EnumerateFiles() => Enumerable.Empty<RelativePath>();
 
+        public IEnumerable<string> GetExpectedPhysicalPath(RelativePath file) =>
```

Comment:
naming: OutputPhysicalPaths?

Reply:
It is expected input file path.

---
In reply to: [140129887](https://github.com/dotnet/docfx/pull/2069#discussion_r140129887) [](ancestors = 140129887)